en/complete_vehicles/crazyflie21.md

```diff
@@ -0,0 +1,252 @@
+The [C based cfbridge](https://github.com/dennisss/cfbridge) is currently experiencing data loss issues, which is why we have chosen to use **cfbridge.py**.
+:::
+
+- Make sure you have set the udev permissions to use the USB Radio. To do this, follow the steps listed [here](https://github.com/bitcraze/crazyflie-lib-python#setting-udev-permissions) and **restart** your computer.
+- Connect a Crazyradio PA via USB.
+- Build a [virtual environment (local python environment)](https://virtualenv.pypa.io/en/latest/) with package dependencies using the following method:
+    ```
+    pip install tox --user
+    ```
+- Navigate to the crazyflie-lib-python folder and type: 
+    ```
+    make venv
+    ```
+- Activate the virtual environment: 
+    ```
+    source venv/bin/activate
+    ```
+- Install required dependencies:
+    ```
+    pip install cflib
+    pip install -r requirements.txt
+    ```
+
+To connect Crazyflie 2.1 with crazyradio, **launch cfbridge** by following these steps:
+- Power off and power on Crazyflie 2.1 and wait for it to boot up.
+- Connect a Crazyflie radio device via USB.
+- Navigate to the crazyflie-lib-python folder.
+- Activate the environment: 
+    ```
+    source venv/bin/activate
+    ```
+- Navigate to the examples folder:
+    ```
+    cd examples
+    ```
+- Launch cfbridge: 
+    ```
+    python cfbridge.py
+    ```
+	
+  :::note
+  *Cfbridge* by default tries to initiate the radio link communication on channel 80 and with crazyflie address 0xE7E7E7E7E7. 
+  If you are using [multiple crazyflies and/or crazyradios](https://github.com/dennisss/cfbridge/blob/master/README.md#advanced-swarming) in the same room and want to use a different channel and/or address for each, first connect the crazyflie with QGroundControl via a USB cable and change the syslink parameters (channel, address) in QGroundControl.
+  Next, launch the cfbridge by giving the same channel and address as   the first and second arguments respectively, e.g: `python cfbridge.py 90 0x0202020202`
+  :::
+- Open QGroundControl.
+- After using *cfbridge*, you can deactivate the virtualenv if you activated it by pressing `CTRL+z`.
+  Most of the time, launching *cfbridge* again from the same terminal doesn't connect to crazyflie, this can be solved by closing the terminal and relaunching *cfbridge* in a new terminal. 
+
+:::tip
+If you change any driver in [crazyflie-lib-python](https://github.com/bitcraze/crazyflie-lib-python) or if launching *cfbridge* in a new terminal does not find crazyflie, you can try navigating to the crazyflie-lib-python folder and run the script below to rebuild cflib.
+```
+make venv
+```
+:::
+
+:::note
+The Joystick menu in QGC, only appears after you connect the controller to the PC (for example a Playstation 3 controller).
+:::
+
+![](../../assets/flight_controller/crazyflie21/joystick_menu_qgc.png)
+
+## Hardware Setup
+
+Crazyflie 2.1 is only able to fly in [Stabilized mode](../flight_modes/manual_stabilized_mc.md). 
```

Comment:
This line should be up in the introduction (near top of page as a warning). I would refer to it as Manual mode.

Reply:
Manual/Stabilized mode would probably be the most complete option. Considering that they are the same mode in MC mode and usually used interchangeably.

Reply:
Totaly! Done.

Reply:
Thanks. FWIW I prefer "manual mode" because I hate typing unnecessarily and because it is not ambiguous. I wouldn't change it though if I encountered it because it is "most correct".